lib/rules/declaration-block-no-shorthand-property-overrides/__tests__/index.mjs

```diff
@@ -47,6 +47,46 @@ testRule({
 			endLine: 1,
 			endColumn: 32,
 		},
+		{
+			code: 'a { border-block: 1px solid; border: 20px dashed black; }',
+			message: messages.rejected('border', 'border-block'),
+			line: 1,
+			column: 30,
+			endLine: 1,
+			endColumn: 36,
+		},
+		{
+			code: 'a { border-block-end: 1px solid; border: 20px dashed black; }',
+			message: messages.rejected('border', 'border-block-end'),
+			line: 1,
+			column: 34,
+			endLine: 1,
+			endColumn: 40,
+		},
+		{
+			code: 'a { border-block-start: 1px solid; border: 20px dashed black; }',
+			message: messages.rejected('border', 'border-block-start'),
+			line: 1,
+			column: 36,
+			endLine: 1,
+			endColumn: 42,
+		},
```

Comment:
These are incorrect and have to be removed.
You could add them as passing if you want to detect regressions though.